packages/connect-react/src/index.ts

```diff
@@ -1,7 +1,139 @@
-export function useAragon(): void {
-  return
+import React, { useContext, useEffect, useMemo, useRef, useState } from 'react'
+import {
+  Application,
+  ConnectOptions,
+  ConnectorDeclaration,
+  Organization,
+  Permission,
+  connect,
+} from '@aragon/connect'
+
+type ContextValue = {
+  org: Organization | null
+  orgError: Error | null
+  orgLoading: boolean
```

Comment:
Wondering if we should update the `org` types to `Organization` as discussed offline regarding naming?

Reply:
I was thinking that we could keep using `org` internally and for variables in examples since it’s a very long word, and keep “organization” for anything that gets exposed (methods, types). What do you think?

Reply:
That makes sense. :) We just have to make sure to be very consistent with this separation though!